src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCounter = 0;
+
+    public static void printHorizontalLine() {
+        System.out.println("    ——————————————————————————————————————————————————");
+    }
+
+    public static void printTasks(int taskCounter, ArrayList<Task> tasks) {
+        for (int i=0; i<taskCounter; i++) {
+            System.out.println("     " + (i+1) + ". " + tasks.get(i));
+        }
+    }
+
+    public static boolean checkCommand(String[] task) {
+        boolean checkTask = false;
+        boolean checkDescription;
+        if (task[0].equals("todo") || task[0].equals("deadline") || task[0].equals("event")) {
+            checkTask = true;
+        }
+
+        if (!checkTask) {
+            System.out.println("     Oops! I'm sorry, but I don't know what that means. :(");
+            return false;
+        }
+
+        if (task.length==2) {
+            checkDescription = true;
+        } else {
+            System.out.println("     Oops! The description of " + task[0] + " cannot be empty.");
+            return false;
+        }
+
+        return checkDescription;
+    }
+
+    public static void addTask(String command) {
+        String[] task = command.split(" ", 2);
+
+        if (!checkCommand(task)) {
+            return;
+        }
+
+        String type = task[0];
+        int index = task[1].indexOf("/");
+        String description = task[1];
+        String date = null;
+
+        if (index != -1) {
+            description = task[1].substring(0, index-1);
+            date = task[1].substring(index+4);
+        }
+
+        System.out.println("     Got it. I've added this task:");
+
+        switch(type) {
```

Comment:
you might want to leave a space like so to improve readability: 
`switch (type) {`